tools/spice/spice_examples.py

```diff
@@ -60,19 +64,22 @@ def get_attitude_timerange(ck_kernel, id):
 
 
 def _get_particle_velocity(direct_events):
-    """Get the particle velocity in the heliosphere frame.
+    """
+    Get the particle velocity in the heliosphere frame.
 
     Parameters
     ----------
-    direct_events: dict
+    direct_events : iterable
```

Comment:
```suggestion
    direct_events : dict
```

I think this was fine the way it was?